### ISSUE TITLE
Look into using docker (or some other solution) to iron out environment issues

### ISSUE THREAD

**Issue by DSchau** [objectpartners/react-redux-workshop#16]
We (still) almost every time run into issues with people's environments not being configured correctly.

If we could create a docker image that contains required versions for things, that would be pretty neat. We could certainly build the server side stuff so that each user doesn't have to run that in a separate command

**Comment by DSchau** [comment]
Whoops. Still worth doing, I just don't really know all that much about this stuff. I can figure this out unless anyone has any interest in creating an official Docker image?

I feel like we _sorta_ solve our issue here, but then it's also "OK now install Docker to iron out your environment"

"What's Docker?" "How do I do it?" etc. etc.

I feel like being able to use something like CodeSandbox solves this problem just as cleanly and probably a bit more simply.

**Comment by Bolerr** [comment]
Any thought of reaching out to Curt Larson of one of our other dev ops guys on this one?

**Comment by DSchau** [comment]
@Bolerr yeah, that seems like a good idea. I'll do that this afternoon just to gauge quality of the idea 🙃
